azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Respawn time of several ores

## Changes Proposed:
- Update the respawntime of Mithril to 1800 seconds
- Update the respawntime of Thorium to 2700 seconds
- Update the respawntime of those in instances to 604800 seconds

## Issues Addressed:

- This addresses an issue where several people camp nodes on chromiecraft with alt accounts.

## SOURCE:

Source: https://github.com/brotalnia/database/blob/master/world_full_12_november_2020.7z

## Tests Performed:

- Applied SQL
- Checked respawn times


## How to Test the Changes:


1. Apply the SQL
2. Check the IDs in the DB

## Known Issues and TODO List:


- The nodes still need pooling but this will prevent excessive abuse for now.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
